support-frontend/assets/pages/contributions-landing/components/ContributionForm.tsx

```diff
@@ -377,6 +406,23 @@ function ContributionForm(props: PropTypes): JSX.Element {
 				)}
 			</div>
 
+			{props.benefitsMessagingAbTestBulletVariant && !isAUDCountryGroup && (
```

Comment:
The `!isAUDCountryGroup` check is also made in `shouldShowBenefitsMessaging()` so I'm not sure we also need to check it again here as well?